README.rst

```diff
@@ -434,6 +434,11 @@ Build and publish the new version:
 
 Changelog
 =========
+2.7
+---
```

Comment:
I think it's normal to only add version numbers at release time - at least that's my preference.

Reply:
:+1: Removed.